### PR TITLE
Include renamed files in the commit summary changed files tooltip

### LINKED ISSUE #15155: UI Hover Bubble glitch when no text
### I hovered over a file's changes stat with no changes and noticed this
Screenshot (2)

### Release version

3.0.5 (x64)

### Operating system

Windows 10

### Steps to reproduce the behavior

1. Add a empty file
2. Hover over additions/deletions
3. mouse may block it, move mouse over to see it.

### Log files

_No response_

### Screenshots

_No response_

### Additional context

This is really minor.

### PR DESCRIPTION
**Pull request by haykam821**
Fixes #15155

### Screenshots

Commit summary '61 changed files' text with tooltip listing three lines: '6 added', '54 modified', and '1 renamed'

### Release notes

Notes:

[Fixed] Include renamed files in the commit summary changed files tooltip

### CODE REVIEW

**Comment by Cervantes12** [comment]
Update to administration main management CEO owner

**Review by tidy-dev — DISMISSED** [review]
@haykam821 This works great! I have one small ask in light of the ticket that originated this PR. Could you add a boolean that simply checks if any of the file kinds we are looking to describe exist? 

Something like:
```
const hasTooltip = check each type count > 0
const filesLongDescription = fileCount > 0 && hasToltip ? make content : undefined
(just pass filesLongDescription as the tooltip property)
```
I am pretty sure adding the rename will make it so we don't have the issue, but it would be nice to safe guard against unknown other scenarios and only show the tooltip if we know we have content to show.

**Comment by tidy-dev** [comment]
@haykam821 Just checking back on this - would love to get it merged. Are you planning on addressing the requested change?

File: app/src/ui/history/commit-summary.tsx [hunk 1]
```diff
@@ -547,9 +548,14 @@ export class CommitSummary extends React.Component<
         case AppFileStatusKind.Deleted:
           filesRemoved += 1
           break
+        case AppFileStatusKind.Renamed:
+          filesRenamed += 1
       }
     }
 
+    const hasFileDescription =
+      filesAdded + filesModified + filesRemoved + filesRemoved > 0
```
Comment (sergiou87) [inline app/src/ui/history/commit-summary.tsx]: I think this should be…
```suggestion
      filesAdded + filesModified + filesRemoved + filesRenamed > 0
```

**Review by sergiou87 — CHANGES_REQUESTED** [review]

**Review by sergiou87 — APPROVED** [review]
Code LGTM and it works like a charm!

**Pull request merged**